app/models/assignment.rb

```diff
@@ -7,6 +7,9 @@ class Assignment < ActiveRecord::Base
     rubric: 'rubric'
   }
 
+  MIN_PEER_REVIEWS_PER_PERSON = 1
+  MAX_PEER_REVIEWS_PER_PERSON = 5
```

Comment:
Do you think these are needed? I don't know if in the future we want to constrain the admins (though currently they can just enter whatever value in the input field regardless of the above) and remove this, or keep it and use it for something later on?